presidio-analyzer/conf/analyzer.yaml

```diff
@@ -0,0 +1,93 @@
+recognizer_registry:
+  recognizers: 
+    - name: CreditCardRecognizer
+      supported_languages: 
+      - en
```

Comment:
This structure is different from the one we use for custom. Here it's `{"supported_languages": ["en", "es"]}` and there it's `{"supported_languages": {"language": "es", "context": [...]}`. Can we / should we have the same structure? I think it's also aligned with what we agreed on the call, as @SharonHart gave this example structure:
```json
[
{
  "name": "CreditCardRecognizer",
  "languages": [
    {"langauge": "es", "context": ["tarjeta","credito"]},
    {"langauge": "en", "context": ["credit"]}
    ],
	"type": "PREDEFINED"
},
  {
  "name": "CreditCardRecognizer",
    "patterns": ["/b/s*[1]/b"],
  "languages": [
    {"language": "es", "context": ["tarjeta","credito"]},
    {"language": "en", "context": ["credit"]}
    ]
    ,"type": "CUSTOM",
    "enabled": false
}
]
```

Reply:
Oh yeah that's just by accident, forgot to change the structure of the predefined, thanks!

Reply:
@RoeyBC this still has the old structure, doesn't it?

Reply:
custom supports the following:
1. old version
2. new version with context:
      supported_languages:
        - language: en
          context: [credit]
        - language: es
          context: [tarjeta,credito]
3. new version without context (in which case it'll be just a string list):
  supported_languages: 
    - en
    - es